File: app/src/test/java/com/nicholasworkshop/tinklabstest/external/content/model/StoryFaker.java
package com.nicholasworkshop.tinklabstest.external.content.model;

import com.github.javafaker.Faker;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by nicholas.wong on 2017/05/16.
 */
public class StoryFaker {

    public static Story fakeStory() {
        Faker faker = new Faker();
        return Story.builder()
                .title(faker.name().name())
                .summary(faker.shakespeare().asYouLikeItQuote())
                .featureImageUrl(faker.internet().image())
                .build();
    }

    public static List<Story> fakeStoryList(int count) {
        List<Story> list = new ArrayList<>();
        for (int i = 0; i < count; i++) {
            list.add(fakeStory());
        }
        return list;
    }
}

File: app/src/test/kotlin/com/nicholasworkshop/tinklabstest/external/content/model/StoryFaker.kt
package com.nicholasworkshop.tinklabstest.external.content.model

import com.github.javafaker.Faker
import java.util.*

object StoryFaker {

    fun fakeStory(): Story {
        val faker = Faker()
        return Story(
                title = faker.name().name(),
                summary = faker.shakespeare().asYouLikeItQuote(),
                featureImageUrl = faker.internet().image())
    }

    fun fakeStoryList(count: Int): List<Story> {
        val list = ArrayList<Story>()
        for (i in 0 until count) {
            list.add(fakeStory())
        }
        return list
    }
}